architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,20 @@ endif::[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+ifdef::openshift-online[]
+[[projects-hibenation]]
+=== Project Hibernation
+In {product-title} Starter, a project exceeding 54 cumulative quota-hours of
+usage in a rolling 72-hour period must hibernate for the next 18 hours.
```

Comment:
in line 142, change to 'must sleep' instead of 'must hibernate'?   
Then, also add something like, 'Also, a project that has been inactive for more than 30 minutes will be idled.  When a project's network activity falls below a configured threshold, a project is deemed inactive.  When a project is idled, the replica count is set to 0 and all pods are deleted.  All PVCs and PVs in the project are left untouched.  Upon receiving network traffic, the replica count will be scaled back to whatever it was before being idled.